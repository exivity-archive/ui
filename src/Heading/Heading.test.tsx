import React from 'react'
import renderer from 'react-test-renderer'
import { Heading } from '.'

test('renders Heading without props', () => {
  const component = renderer.create(<Heading />)
  expect(component.toJSON()).toMatchSnapshot()
})

test('renders Heading with text', () => {
  const component = renderer.create(<Heading>Heading</Heading>)
  expect(component.toJSON()).toMatchSnapshot()
})

test('renders Heading with different levels', () => {
  const component1 = renderer.create(<Heading level={1} />)
  expect(component1.toJSON()).toMatchSnapshot()
  const component2 = renderer.create(<Heading level={2} />)
  expect(component2.toJSON()).toMatchSnapshot()
  const component3 = renderer.create(<Heading level={3} />)
  expect(component3.toJSON()).toMatchSnapshot()
})
