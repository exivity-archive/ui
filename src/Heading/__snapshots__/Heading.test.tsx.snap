// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Heading with different levels 1`] = `
.c0 {
  font-family: 'Fira Sans',sans-serif;
  font-weight: normal;
  font-size: 16px;
  color: #222;
  line-height: 1.5;
}

<h1
  className="c0"
/>
`;

exports[`renders Heading with different levels 2`] = `
.c0 {
  font-family: 'Fira Sans',sans-serif;
  font-weight: normal;
  font-size: 16px;
  color: #222;
  line-height: 1.5;
}

<h2
  className="c0"
/>
`;

exports[`renders Heading with different levels 3`] = `
.c0 {
  font-family: 'Fira Sans',sans-serif;
  font-weight: normal;
  font-size: 16px;
  color: #222;
  line-height: 1.5;
}

<h3
  className="c0"
/>
`;

exports[`renders Heading with text 1`] = `
.c0 {
  font-family: 'Fira Sans',sans-serif;
  font-weight: normal;
  font-size: 16px;
  color: #222;
  line-height: 1.5;
}

<h1
  className="c0"
>
  Heading
</h1>
`;

exports[`renders Heading without props 1`] = `
.c0 {
  font-family: 'Fira Sans',sans-serif;
  font-weight: normal;
  font-size: 16px;
  color: #222;
  line-height: 1.5;
}

<h1
  className="c0"
/>
`;
